.jscsrc

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
I'm afraid that it won't work on windows. I'd suggest here to use `{ "preset": "jscs" }`. ;-)